Fibonachi.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class Fibonachi {
+  public static void main(String[] args) {
+    final int INT_LENGHT = 2147483647;
```

Comment:
Invalid name - it's max value, not length, anyway - use constant from class  Integer